ressources/db_lhd.sql

```diff
@@ -25,7 +25,7 @@ DROP SCHEMA IF EXISTS lhd CASCADE;
 --
 
 CREATE SCHEMA lhd;
-ALTER DATABASE lhd; SET search_path TO lhd;
+ALTER DATABASE lhd SET search_path TO lhd;
```

Comment:
previous Merge issues

Reply:
err, you should delete this line it is already done at the beginning of the file

Reply:
right, done in another commit